src/fabric-third-party/web/index.js

```diff
@@ -0,0 +1,36 @@
+//@flow
+import { getIframeId, resizeIframeHeight, getWebfonts } from '../../_shared/js/messages';
+import { write } from '../../_shared/js/dom';
+
+const ThirdPartyTag = '[%ThirdPartyTag%]';
```

Comment:
We don't seem to use this const? 

Also JS convention of using lower camel case when naming would probably make this:
`const thirdPartyTag = ...`